test/views/EditSetRecordEditDiscardSpec.scala

```diff
@@ -21,33 +21,31 @@
 
 package views
 
-import org.jsoup.nodes.Document
 import play.api.i18n.Messages
 import play.api.test.Helpers
-import play.twirl.api.Html
 import support.BaseSpec
 import support.CommonMatchers._
+import uk.gov.hmrc.govukfrontend.views.html.components.GovukNotificationBanner
 import uk.gov.nationalarchives.omega.editorial.models.PhysicalRecord
 import uk.gov.nationalarchives.omega.editorial.views.html.editSetRecordEditDiscard
 
 class EditSetRecordEditDiscardSpec extends BaseSpec {
```

Comment:
Given all of the assertions on the contents, I'd suggest that this be renamed to `EditSetRecordEditDiscardViewSpec`.

Reply:
Agreed, I've made that change.

Reply:
Confirmed.